rotkehlchen/tests/data_migrations/test_migrations.py

```diff
@@ -517,3 +524,9 @@ def assert_progress_message(msg, step_num, description) -> None:
             assert_progress_message(msg, i, None)
         else:
             assert_progress_message(msg, i, None)
+
+    # check that the spam assets have been updated
```

Comment:
you would need to add a tiny check for 1 more websocket message here once you do what I asked in the data migration